src/vnsw/agent/port_ipc/port_ipc_handler.cc

```diff
@@ -499,7 +499,7 @@ bool PortIpcHandler::GetPortInfo(const string &uuid_str, string &info) const {
 
 bool PortIpcHandler::InterfaceExists(const std::string &name) const {
 #if 0 //WINDOWS-TEMP
-    int indx  = if_nametoindex(name.c_str());
+    int indx  = osspecific_if_nametoindex(name.c_str());
```

Comment:
As per our Tuesday discussion, we should avoid breaking compatibility if possible.

I believe simply replacing it with `if_nametoindex` would work okay. Am i wrong here?

However, another way to handle it is adding `osspecific_if_nametoindex` for Linux/FreeBSD. This is also okay resolution if you prefer it.